apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/page.tsx

```diff
@@ -130,7 +130,16 @@ export default async function DeploymentPage({
                       value={deployment.jobAgentConfig}
                       jobAgents={jobAgents}
                       workspace={workspace}
-                      deploymentId={deployment.id}
+                      onSubmit={async (data) => {
```

Comment:
I tried doing without the brackets but it became less redable so just kept it with the async await